org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/core/compiler/IProblem.java

```diff
@@ -1877,7 +1877,7 @@ public interface IProblem {
 	/** @since 3.10 */
 	int NullityMismatchingTypeAnnotationSuperHint = Internal + 954;
 	/** @since 3.10 */
-	int NullityUncheckedTypeAnnotationDetail = Internal + 955;
+	int NullityUncheckedTypeAnnotationDetail = Internal + 955; // see also NullityUncheckedTypeAnnotation
```

Comment:
Could we add a comment explaining here the different between this `NullityUncheckedTypeAnnotationDetail` and `NullityUncheckedTypeAnnotation`  ?

Reply:
sure, thanks.